contracts/cw1-whitelist/src/contract.rs

```diff
@@ -16,6 +19,7 @@ use crate::state::{AdminList, ADMIN_LIST};
 const CONTRACT_NAME: &str = "crates.io:cw1-whitelist";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+#[cfg_attr(not(feature = "library"), entry_point)]
```

Comment:
This is proper, and should be the default for all, whether or not se currently import them as libraries, they should be importable as libraries.